src/util.py

```diff
@@ -24,3 +30,33 @@ def make_duplicates_unique(names_with_duplicates):
             names_without_duplicates[i] += f" ({count})"
 
     return names_without_duplicates
+
+
+def translate_dishes(menus: Dict[date, Menu], language: str) -> bool:
+    """
+    Translate the dish titles of a menu
+
+    :param menus: Menus dictionary as given by the menu parser, will be modified
+    :param language: identifier for a language
+    :return: Whether translation was successful
+    """
+    # get api key from environment, abort if not given
+    deepl_api_key = os.environ.get("DEEPL_API_KEY_EAT_API")
+    if deepl_api_key is None:
+        raise Exception("For translation please provide a DeepL api key via DEEPL_API_KEY_EAT_API")
+
+    translator = deepl.Translator(deepl_api_key)
+    # source language is always german
+    source_language = "de"
```

Comment:
```suggestion
    source_language = "DE"
```

Reply:
I have updated the languages accordingly